voice-client-javascript/assets/admin/actions.private.js

```diff
@@ -75,13 +101,34 @@ class Actions {
 
   async setCallerId({ number }) {
     return {
-      CALLER_ID: number
+      CALLER_ID: number,
+    };
+  }
+
+  async clearCallerId() {
```

Comment:
When I call these from the UI, it is done dynamically. As I am a async/await n00b, do you know what happens if I `await` a non-async function? I was considering asynchronous behavior as part of the "loose Interface" of an Action (a function that does something on your behalf and returns a dictionary of environment variable name to value).

Reply:
since these aren't truly async functions, if you rewrote the method to be a standard function, then you can also remove the need to call it with `await`. 

The pattern makes sense for `chooseLogicalCallerId` but just because one of the functions uses the asyc/await behavior, that doesn't mean everything needs to have the same interface

Reply:
Oh sorry I should've been more clear, I meant it's called generically here:

https://github.com/craigsdennis/function-templates/blob/main/voice-client-javascript/functions/admin/perform-action.js#L24

I can just see what happens if I pop a regular function through...I'm assuming I got burned originally.